docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -1,39 +1,65 @@
 ////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
+This guide is maintained in the main Quarkus repository.
+To contribute, submit a pull request here:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Using OpenID Connect (OIDC) and Keycloak to centralize authorization
 include::_attributes.adoc[]
 :diataxis-type: howto
 :categories: security
-:keywords: sso oidc security keycloak
-:topics: security,authentication,authorization,keycloak,sso,oidc
-:extensions: io.quarkus:quarkus-oidc,io.quarkus:quarkus-keycloak-authorization
+:keywords: sso, oidc, security, keycloak
+:topics: security, authentication, authorization, keycloak, sso, oidc
+:extensions: io.quarkus:quarkus-oidc, io.quarkus:quarkus-keycloak-authorization
 
-Learn how to enable bearer token authorization in your Quarkus application using link:https://www.keycloak.org/docs/latest/authorization_services/index.html[Keycloak Authorization Services] for secure access to protected resources.
+Learn how to enable bearer token authorization in your Quarkus application by using link:https://www.keycloak.org/docs/latest/authorization_services/index.html[Keycloak Authorization Services] for secure access to protected resources.
 
-The `quarkus-keycloak-authorization` extension relies on `quarkus-oidc`.
-It includes a policy enforcer that regulates access to secured resources.
-Access is governed by permissions set in Keycloak.
-Currently, this extension is compatible solely with Quarkus xref:security-oidc-bearer-token-authentication.adoc[OIDC service applications].
+== Overview
 
-It provides a flexible and dynamic authorization capability based on Resource-Based Access Control.
+The `quarkus-keycloak-authorization` extension builds on the `quarkus-oidc` extension to offer advanced authorization capabilities. It includes a policy enforcer that dynamically regulates access to secured resources. Access is governed by permissions defined in Keycloak, supporting flexible and dynamic Resource-Based Access Control (RBAC).
 
-Rather than explicitly enforcing access through specific mechanisms such as role-based access control (RBAC), `quarkus-keycloak-authorization` determines request permissions based on resource attributes such as name, identifier, or Uniform Resource Identifier (URI).
-This process involves sending a `quarkus-oidc`-verified bearer access token to Keycloak Authorization Services for an authorization decision.
+Use this extension if:
 
-Use `quarkus-keycloak-authorization` only if you work with Keycloak and have Keycloak Authorization Services enabled to make authorization decisions.
-Use `quarkus-oidc` if you do not work with Keycloak or work with Keycloak but do not have its Keycloak Authorization Services enabled to make authorization decisions.
+- You are using Keycloak.
```

Comment:
IMHO both sentences should be connected with `and`

Reply:
Updated. Thanks.